bins/packages/cloudhypervisor/cloudhypervisor.sh

```diff
@@ -0,0 +1,62 @@
+CLOUDHYPERVISOR_VERSION="0.13.0"
+CLOUDHYPERVISOR_CHECKSUM="44bd96c255b8d573a83eef2364d934ae"
+CLOUDHYPERVISOR_LINK="https://github.com/cloud-hypervisor/cloud-hypervisor/archive/v${CLOUDHYPERVISOR_VERSION}.tar.gz"
+CLOUDHYPERVISOR_RUST_TOOLCHAIN="1.48.0"
```

Comment:
Is there any reason we are building with this version and not latest stable (currently 1.50)?

Reply:
no